app/mailers/feed_invitation_mailer.rb

```diff
@@ -1,11 +1,14 @@
 class FeedInvitationMailer < ApplicationMailer
   layout nil
 
-  def notify(record)
+  def notify(record_id, team_id)
+    record = FeedInvitation.find_by_id record_id
+    team = Team.find_by_id team_id
     @recipient = record.email
     @user = record.user
     @feed = record.feed
     @due_at = record.created_at + CheckConfig.get('feed_invitation_due_to', 30).to_i.days
+    @accept_feed_url = "#{CheckConfig.get('checkdesk_client')}/#{team&.slug}/feed/#{record.id}/accept_invitation"
```

Comment:
@melsawy I suggest to remove the `&` - the link won't work without a team, anyway.